scripts/feature-init.ts

```diff
@@ -81,6 +81,7 @@ async function main() {
     process.exit(0);
   }
   await fs.writeFile(destination, formatted);
+  console.log(`Wrote ${destination}`);
```

Comment:
If you write out just the file name, then it's possible to do something like this: `my-editor $(npm run --silent feature-int -- …)` or pipe it to another command, right?

```suggestion
  console.log(destination);
```